src/index.native.scroll.js

```diff
@@ -103,7 +103,9 @@ class SwipeableViews extends React.Component {
       index,
     } = nextProps;
 
-    if (typeof index === 'number' && index !== this.props.index) {
+    // Sometimes the index can be the same, but the slide
+    // may have changed, so just always set it
+    if (typeof index === 'number') {
```

Comment:
I have added this commit to master https://github.com/oliviertassinari/react-swipeable-views/commit/c7768d8f2a7bd75e2d5dcf7e193a2e61c0848eef.
Regarding this change, could it be because of unwanted upgrade from `handleMomentumScrollEnd` or `handleLayout`?